Ref/src/adapter/eps/controller/component/epsControllerComponent.cpp

```diff
@@ -0,0 +1,109 @@
+// acknowledged.
+//
+// ======================================================================
+
+#include <Ref/src/adapter/eps/controller/component/epsControllerComponent.hpp>
+
+#include "Fw/Types/BasicTypes.hpp"
+
+namespace Ref {
+
+// ----------------------------------------------------------------------
+// Construction, initialization, and destruction
+// ----------------------------------------------------------------------
+
+epsControllerComponent ::
+    epsControllerComponent(
+        const char *const compName) : epsControllerComponentComponentBase(compName) {
+}
+
+void epsControllerComponent ::
+    init(
+        const NATIVE_INT_TYPE queueDepth,
+        const NATIVE_INT_TYPE instance) {
+    epsControllerComponentComponentBase::init(queueDepth, instance);
+}
+
+epsControllerComponent ::
+    ~epsControllerComponent() {
+}
+
+// ----------------------------------------------------------------------
+// Handler implementations for user-defined typed input ports
+// ----------------------------------------------------------------------
+
+void epsControllerComponent ::
+    recvBatteryFromEPSIn_handler(
+        const NATIVE_INT_TYPE portNum,
+        F32 battery_level) {
+    this->sendBatteryToStateOut_out(portNum, battery_level);
+}
+
+void epsControllerComponent ::
+    recvBatteryFromStateIn_handler(
+        const NATIVE_INT_TYPE portNum,
+        F32 battery_level) {
+    this->sendBatteryToConopsOut_out(portNum, battery_level);
+}
+
+void epsControllerComponent ::
+    recvBatteryReqFromConopsIn_handler(
+        const NATIVE_INT_TYPE portNum,
+        bool req_status) {
+    this->reqBatteryFromStateOut_out(portNum, req_status);
+}
+
+void epsControllerComponent ::
+    recvPwrCmdFromConopsIn_handler(
+        const NATIVE_INT_TYPE portNum,
+        bool power) {
+    this->sendPwrCmdToEPSOut_out(portNum, power);
+}
+
+void epsControllerComponent ::
+    recvPwrStatusFromEPSIn_handler(
+        const NATIVE_INT_TYPE portNum,
+        bool status) {
+    this->sendPwrStatusOut_out(portNum, status);
+}
+
+// ----------------------------------------------------------------------
+// Command handler implementations
+// ----------------------------------------------------------------------
+
+void epsControllerComponent ::
+    SEND_BATTERY_COMMAND_cmdHandler(
+        const FwOpcodeType opCode,
+        const U32 cmdSeq,
+        Ref::InternalType c) {
+    // TODO
+    this->cmdResponse_out(opCode, cmdSeq, Fw::CmdResponse::OK);
+}
+
+void epsControllerComponent ::
+    GET_STATE_cmdHandler(
+        const FwOpcodeType opCode,
+        const U32 cmdSeq,
+        Ref::InternalType c) {
+    // TODO
+    this->cmdResponse_out(opCode, cmdSeq, Fw::CmdResponse::OK);
+}
+
+void epsControllerComponent ::
+    SEND_DATA_CONOPS_cmdHandler(
+        const FwOpcodeType opCode,
+        const U32 cmdSeq,
+        Ref::InternalType c) {
+    // TODO
+    this->cmdResponse_out(opCode, cmdSeq, Fw::CmdResponse::OK);
+}
```

Comment:
Based on discussions with @NishqR I don't think any of our components should have commands.

Commands should be carefully designed and added in later down the line, we want to reduce entry points to the system and carefully control where changes can be made

Reply:
yeah no commands for now - at least not in these components, if we do ever end up having commands from ground we'll have to think of a more centralized way to handle them. 

But yeah we should probably get rid of these - I'm guilty of having them in some other branches too (just for exploring and testing functionality) 

Not sure if they can just be simply yeeted tho (as in lines deleted from all files) without throwing other errors

Reply:
nvm can confirm they can just be yeeted

Reply:
just need to make sure to remove em from .fpp, .cpp and .hpp files @shqln

Reply:
yeah yeeting causing errors when I try to run the unit tests again - errors aren't exactly clear